### DISCUSSION TITLE
Could we exploit the Set up with Android Phone option?

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1262] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
We might be able to unenroll using that

**Comment by HelloIloveGimkit2** [comment, marked as answer]
no you wouldn't your best shot is to use br1ck and unenroll that way but you can unenroll using the android phone option cuz enrollment is stored in the VPD, so in short terms,  you can't cuz it's stored on the vpd (vital product data) and you have to figure out a way to access it just use br1ck at that point
